lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -134,7 +135,7 @@ def process_body_from_alb_invocation(event):
 
     return [event_body]
 
-
+@lru_cache(maxsize=None)
```

Comment:
Was this tested?

Reply:
I added this line.  I didn't know of a way to unit test it so I just sent 10 messages through (and forced them to fail so that they would be replayed in quick succession once coming off the queue).  If you look at the [logs for these records](https://console.amazonaws-us-gov.com/cloudwatch/home?region=us-gov-west-1#logsV2:log-groups/log-group/$252Faws$252Flambda$252Fproject-dev-vetext-incoming-forwarder-lambda/log-events/2022$252F09$252F27$252F$255B$2524LATEST$255Da04c47feefe44b9280b306454829073b), you'll see that it logs out "retrieved authtoken from ssm" but does NOT log "Generated ssm_client".  This shows that the cache was used to retrieve the value because it didnt generate the boto3 ssm client in order to retrieve the value.  

I verified that the request that is made includes the basic auth token header with the proper value that is stored in parameter store.

![image](https://user-images.githubusercontent.com/4851874/192614110-45a51904-a562-43e7-b761-d6def20b31e7.png)